docs/troubleshooting.asciidoc

```diff
@@ -28,3 +29,110 @@ include::copied-from-beats/getting-help.asciidoc[]
 == Debug
 
 include::copied-from-beats/debugging.asciidoc[]
+
+[[common-problems]]
+== Common problems
+
+This section describes common problems you might encounter with APM Server.
+
+[[queue-is-full]]
+=== 503: Queue is full
+
+APM Server has an internal queue that buffers documents until they can be delivered to Elasticsearch.
+The internal queue helps to: 
+
+* alleviate problems that might occur if Elasticsearch is intermittently unavailable
+* handle large spikes of data arriving at the APM Server at the same time
+* send documents to Elasticsearch in bulk, instead of individually
```

Comment:
```diff
-* send documents to Elasticsearch in bulk, instead of individually
+* send documents to Elasticsearch in bulk,
+  instead of individually
```